cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityAdministrationTest.scala

```diff
@@ -980,16 +981,26 @@ class SecurityAdministrationTest extends DocumentingTest with QueryStatisticsTes
           p(
             """The name-globbing for procedure and user defined function names is a simplified version of globbing for filename expansions, only allowing two wildcard characters -- `+*+` and `?`.
               |They are used for multiple and single character matches, where `+*+` means 0 or more characters and `?` matches exactly one character.""".stripMargin)
+          note {
+            p("""The name-globbing is subject to the <<cypher-manual#cypher-naming, standard Cypher restrictions on valid identifiers>>,
+              |with the exception that it may include dots, stars, and question marks without the need for escaping using backticks.
+              |Each part of the name-globbing separated by dots may be individually escaped, for example `++mine.`procedureWith%`++` but not `++mine.procedure`With%`++`.
+              |Also good to keep in mind is that the wildcard characters will still behave as wildcards even when escaped,
+              |as an example using `+*+` or `++`*`++` both allow executing all functions or procedures and not all vs only the procedure or function named `+*+`.
```

Comment:
I find this part hard to read/understand. Maybe it can be simplified to something like:

`` as an example, using `++`*`++` is equivalent to using `+*+` and thus allows executing all functions or procedures and not only the procedure or function named `+*+`. ``